protocol-gateway/azure-mqtt-protocol-gateway/README.md

```diff
@@ -0,0 +1,166 @@
+The default configuration can be found in the file `protocol-gateway/azure-mqtt-protocol-gateway/src/main/resources/application.properties` 
+and can be customized using [Spring Boot Configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config). 
+ 
+ 
+### Starting a Receiver
+
+Telemetry and event messages need an application that consumes the messages. 
+The [Getting started](https://www.eclipse.org/hono/getting-started/#starting-the-example-application) guide shows how to start the example application that receives the messages.
+ 
+ 
+## Starting the Protocol Gateway
+
+Build the template project:
+~~~sh
+# in directory: protocol-gateway/mqtt-protocol-gateway-template/
+mvn clean install
+~~~
+
+and start the protocol gateway:
+~~~sh
+# in directory: protocol-gateway/azure-mqtt-protocol-gateway/
+mvn spring-boot:run
+~~~
+
+ 
+## Enable TLS 
+
+Azure IoT Hub only provides connections with TLS and only offers port 8883. To start the protocol gateway listening
+on this port with TLS enabled and a demo certificate, run:
+
+~~~sh
+# in directory: protocol-gateway/azure-mqtt-protocol-gateway/
+mvn spring-boot:run -Dspring-boot.run.profiles=ssl
+~~~
+**NB** Do not forget to build the template project before, as shown above.
+
+With the [Eclipse Mosquitto](https://mosquitto.org/) command line client, for example, sending an event message would then look like this:
+
+~~~sh
+# in directory: protocol-gateway/azure-mqtt-protocol-gateway
+mosquitto_pub -d -h localhost -p 8883 -i '4712' -u 'demo1' -P 'demo-secret'  -t "devices/4712/messages/events/" -m "hello world" -V mqttv311 --cafile target/config/hono-demo-certs-jar/trusted-certs.pem
+~~~
+
+Existing hardware devices might need to be configured to accept the used certificate. 
+
+## Example Requests
+
+With the [Eclipse Mosquitto](https://mosquitto.org/) command line client the requests look like the following.
+ 
+**Telemetry**
+ 
+~~~sh
+mosquitto_pub -d -h localhost -i '4712' -u 'demo1' -P 'demo-secret' -t 'devices/4712/messages/events/?foo%20bar=b%5Fa%5Fz' -m "hello world" -V mqttv311 -q 0
+~~~
+ 
+**Events**
+ 
+~~~sh
+mosquitto_pub -d -h localhost -i '4712' -u 'demo1' -P 'demo-secret' -t 'devices/4712/messages/events/?foo%20bar=b%5Fa%5Fz' -m '{"alarm": 1}' -V mqttv311 -q 1
+~~~
+ 
+### Commands 
+
+The example application can be used to send commands. 
+The [Getting started](https://www.eclipse.org/hono/getting-started/#advanced-sending-commands-to-a-device) shows a walk-through example.
+
+**Subscribe for one-way commands**
+ 
+~~~sh
+mosquitto_sub -v -h localhost -u "demo1" -P "demo-secret" -t 'devices/4712/messages/devicebound/#' -q 1
+~~~
+ 
+**Subscribe for request-response commands**
+ 
+~~~sh
+mosquitto_sub -v -h localhost -u "demo1" -P "demo-secret" -t '$iothub/methods/POST/#' -q 1
+~~~
+
+When Mosquitto receives the command, in the terminal should appear something like this:
```

Comment:
Instead of "in the terminal should appear something like this", I would suggest "the output in the terminal should look like this".